Comparing source compatibility of /home/someone/git/com.github/io7m/jregions/com.io7m.jregions.generators/target/com.io7m.jregions.generators-0.0.4.jar against /home/someone/.m2/repository/com/io7m/jregions/com.io7m.jregions.generators/0.0.2/com.io7m.jregions.generators-0.0.2.jar
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaBDGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaBDGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaBDGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.areas.AreaBD next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaBIGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaBIGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaBIGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.areas.AreaBI next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaDGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaDGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaDGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.areas.AreaD next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaFGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaFGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaFGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.areas.AreaF next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaIGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaIGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaIGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.areas.AreaI next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaLGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaLGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaLGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.areas.AreaL next()
+++  NEW CLASS: PUBLIC(+) FINAL(+) com.io7m.jregions.generators.AreaSizeBDGenerator  (not serializable)
	+++  NEW INTERFACE: net.java.quickcheck.Generator
	+++  NEW CONSTRUCTOR: PUBLIC(+) AreaSizeBDGenerator(net.java.quickcheck.Generator)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.generators.AreaSizeBDGenerator create()
	+++  NEW METHOD: PUBLIC(+) com.io7m.jregions.core.unparameterized.sizes.AreaSizeBD next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaSizeBIGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaSizeBIGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaSizeBIGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.sizes.AreaSizeBI next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaSizeDGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaSizeDGenerator(net.java.quickcheck.generator.support.DoubleGenerator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaSizeDGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.sizes.AreaSizeD next()
+++  NEW CLASS: PUBLIC(+) FINAL(+) com.io7m.jregions.generators.AreaSizeFGenerator  (not serializable)
	+++  NEW INTERFACE: net.java.quickcheck.Generator
	+++  NEW CONSTRUCTOR: PUBLIC(+) AreaSizeFGenerator(net.java.quickcheck.generator.support.DoubleGenerator)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.generators.AreaSizeFGenerator create()
	+++  NEW METHOD: PUBLIC(+) com.io7m.jregions.core.unparameterized.sizes.AreaSizeF next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaSizeIGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaSizeIGenerator(net.java.quickcheck.generator.support.IntegerGenerator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaSizeIGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.sizes.AreaSizeI next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.AreaSizeLGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC AreaSizeLGenerator(net.java.quickcheck.generator.support.LongGenerator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.AreaSizeLGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.unparameterized.sizes.AreaSizeL next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaBDGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaBDGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaBDGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.areas.PAreaBD next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaBIGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaBIGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaBIGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.areas.PAreaBI next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaDGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaDGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaDGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.areas.PAreaD next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaFGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaFGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaFGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.areas.PAreaF next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaIGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaIGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaIGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.areas.PAreaI next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaLGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaLGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaLGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.areas.PAreaL next()
+++  NEW CLASS: PUBLIC(+) FINAL(+) com.io7m.jregions.generators.PAreaSizeBDGenerator  (not serializable)
	+++  NEW INTERFACE: net.java.quickcheck.Generator
	+++  NEW CONSTRUCTOR: PUBLIC(+) PAreaSizeBDGenerator(net.java.quickcheck.Generator)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.generators.PAreaSizeBDGenerator create()
	+++  NEW METHOD: PUBLIC(+) com.io7m.jregions.core.parameterized.sizes.PAreaSizeBD next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaSizeBIGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaSizeBIGenerator(net.java.quickcheck.Generator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaSizeBIGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.sizes.PAreaSizeBI next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaSizeDGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaSizeDGenerator(net.java.quickcheck.generator.support.DoubleGenerator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaSizeDGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.sizes.PAreaSizeD next()
+++  NEW CLASS: PUBLIC(+) FINAL(+) com.io7m.jregions.generators.PAreaSizeFGenerator  (not serializable)
	+++  NEW INTERFACE: net.java.quickcheck.Generator
	+++  NEW CONSTRUCTOR: PUBLIC(+) PAreaSizeFGenerator(net.java.quickcheck.generator.support.DoubleGenerator)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.generators.PAreaSizeFGenerator create()
	+++  NEW METHOD: PUBLIC(+) com.io7m.jregions.core.parameterized.sizes.PAreaSizeF next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaSizeIGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaSizeIGenerator(net.java.quickcheck.generator.support.IntegerGenerator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaSizeIGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.sizes.PAreaSizeI next()
===  UNCHANGED CLASS: PUBLIC FINAL com.io7m.jregions.generators.PAreaSizeLGenerator  (not serializable)
	===  UNCHANGED INTERFACE: net.java.quickcheck.Generator
	===  UNCHANGED CONSTRUCTOR: PUBLIC PAreaSizeLGenerator(net.java.quickcheck.generator.support.LongGenerator)
	===  UNCHANGED METHOD: PUBLIC STATIC com.io7m.jregions.generators.PAreaSizeLGenerator create()
	===  UNCHANGED METHOD: PUBLIC com.io7m.jregions.core.parameterized.sizes.PAreaSizeL next()
